### ISSUE TITLE
Failed building PHP 7.0 image

### ISSUE THREAD

**Issue by rdarcy1** [konsulting/porter#63]
Problem with `libgcj17_6.4.0` package.

```
Output:
================
Step 1/9 : FROM ubuntu:18.04
 ---> 7698f282e524
Step 2/9 : RUN apt-get update     && apt-get install -y locales     && locale-gen en_US.UTF-8     && apt-get autoremove -y     && apt-get clean     && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 ---> Using cache
 ---> f3cf6f5378cc
Step 3/9 : ENV DEBIAN_FRONTEND=noninteractive     LANG=en_US.UTF-8     LANGUAGE=en_US:en     LC_ALL=en_US.UTF-8
 ---> Using cache
 ---> cf67813e88f9
Step 4/9 : RUN apt-get update     && apt-get install -y curl zip unzip git software-properties-common     && add-apt-repository -y ppa:ondrej/php     && apt-get install -y            php7.0-bcmath            php7.0-cli            php7.0-curl            php7.0-gd            php7.0-mysql            php7.0-pgsql            php7.0-imap            php7.0-imagick            php7.0-memcached            php7.0-mbstring            php7.0-opcache            php7.0-soap            php7.0-sqlite            php7.0-xdebug            php7.0-xml            php7.0-zip            libfontconfig1 libxrender1            vim     && mkdir /run/php     && apt-get remove -y --purge software-properties-common     && apt-get autoremove -y     && apt-get clean     && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 ---> Using cache
 ---> 86c7182babbc
Step 5/9 : RUN php -r "readfile('http://getcomposer.org/installer');" | php -- --install-dir=/usr/bin/ --filename=composer &&     curl -sSL "https://github.com/mailhog/mhsendmail/releases/download/v0.2.0/mhsendmail_linux_amd64" -o /usr/local/bin/mhsendmail &&     chmod +x /usr/local/bin/mhsendmail
 ---> Using cache
 ---> 5c55e7a3c901
Step 6/9 : RUN sed -i -e "s|xdebug.so|/usr/lib/php/20151012/xdebug.so|" /etc/php/7.0/mods-available/xdebug.ini
 ---> Using cache
 ---> 9ed26c5853f2
Step 7/9 : RUN curl -OL http://mirrors.kernel.org/ubuntu/pool/main/g/gcc-6/libgcj17_6.4.0-8ubuntu1_amd64.deb          -OL http://mirrors.kernel.org/ubuntu/pool/main/g/gcc-defaults/libgcj-common_6.4-3ubuntu1_all.deb          -OL http://mirrors.kernel.org/ubuntu/pool/universe/p/pdftk/pdftk_2.02-4build1_amd64.deb          -OL http://mirrors.kernel.org/ubuntu/pool/universe/p/pdftk/pdftk-dbg_2.02-4build1_amd64.deb     && apt-get update     && apt-get install -y ./libgcj17_6.4.0-8ubuntu1_amd64.deb         ./libgcj-common_6.4-3ubuntu1_all.deb         ./pdftk_2.02-4build1_amd64.deb         ./pdftk-dbg_2.02-4build1_amd64.deb     && rm ./libgcj17_6.4.0-8ubuntu1_amd64.deb         ./libgcj-common_6.4-3ubuntu1_all.deb         ./pdftk_2.02-4build1_amd64.deb          ./pdftk-dbg_2.02-4build1_amd64.deb     && apt-get remove -y --purge software-properties-common         && apt-get autoremove -y         && apt-get clean         && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 ---> Running in 335f00facbd4
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   311  100   311    0     0    901      0 --:--:-- --:--:-- --:--:--   901
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   311  100   311    0     0   1095      0 --:--:-- --:--:-- --:--:--  1095
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   311  100   311    0     0   1079      0 --:--:-- --:--:-- --:--:--  1079
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   311  100   311    0     0   1050      0 --:--:-- --:--:-- --:--:--  1050
Get:1 http://security.ubuntu.com/ubuntu bionic-security InRelease [88.7 kB]
Get:2 http://ppa.launchpad.net/ondrej/php/ubuntu bionic InRelease [20.8 kB]
Get:3 http://archive.ubuntu.com/ubuntu bionic InRelease [242 kB]
Get:4 http://archive.ubuntu.com/ubuntu bionic-updates InRelease [88.7 kB]
Get:5 http://archive.ubuntu.com/ubuntu bionic-backports InRelease [74.6 kB]
Get:6 http://ppa.launchpad.net/ondrej/php/ubuntu bionic/main amd64 Packages [54.1 kB]
Get:7 http://security.ubuntu.com/ubuntu bionic-security/main amd64 Packages [486 kB]
Get:8 http://security.ubuntu.com/ubuntu bionic-security/restricted amd64 Packages [5,436 B]
Get:9 http://security.ubuntu.com/ubuntu bionic-security/universe amd64 Packages [324 kB]
Get:10 http://security.ubuntu.com/ubuntu bionic-security/multiverse amd64 Packages [4,168 B]
Get:11 http://archive.ubuntu.com/ubuntu bionic/restricted amd64 Packages [13.5 kB]
Get:12 http://archive.ubuntu.com/ubuntu bionic/main amd64 Packages [1,344 kB]
Get:13 http://archive.ubuntu.com/ubuntu bionic/universe amd64 Packages [11.3 MB]
Get:14 http://archive.ubuntu.com/ubuntu bionic/multiverse amd64 Packages [186 kB]
Get:15 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 Packages [826 kB]
Get:16 http://archive.ubuntu.com/ubuntu bionic-updates/universe amd64 Packages [1,213 kB]
Get:17 http://archive.ubuntu.com/ubuntu bionic-updates/restricted amd64 Packages [10.8 kB]
Get:18 http://archive.ubuntu.com/ubuntu bionic-updates/multiverse amd64 Packages [7,236 B]
Get:19 http://archive.ubuntu.com/ubuntu bionic-backports/main amd64 Packages [2,496 B]
Get:20 http://archive.ubuntu.com/ubuntu bionic-backports/universe amd64 Packages [3,902 B]
Fetched 16.3 MB in 3s (5,131 kB/s)
Reading package lists...
Reading package lists...
E: Sub-process Popen returned an error code (2)
E: Encountered a section with no Package: header
E: Problem with MergeList /libgcj17_6.4.0-8ubuntu1_amd64.deb
E: The package lists or status file could not be parsed or opened.


Error Output:
================
Building php_cli_7-0
Service 'php_cli_7-0' failed to build: The command '/bin/sh -c curl -OL http://mirrors.kernel.org/ubuntu/pool/main/g/gcc-6/libgcj17_6.4.0-8ubuntu1_amd64.deb          -OL http://mirrors.kernel.org/ubuntu/pool/main/g/gcc-defaults/libgcj-common_6.4-3ubuntu1_all.deb          -OL http://mirrors.kernel.org/ubuntu/pool/universe/p/pdftk/pdftk_2.02-4build1_amd64.deb          -OL http://mirrors.kernel.org/ubuntu/pool/universe/p/pdftk/pdftk-dbg_2.02-4build1_amd64.deb     && apt-get update     && apt-get install -y ./libgcj17_6.4.0-8ubuntu1_amd64.deb         ./libgcj-common_6.4-3ubuntu1_all.deb         ./pdftk_2.02-4build1_amd64.deb         ./pdftk-dbg_2.02-4build1_amd64.deb     && rm ./libgcj17_6.4.0-8ubuntu1_amd64.deb         ./libgcj-common_6.4-3ubuntu1_all.deb         ./pdftk_2.02-4build1_amd64.deb          ./pdftk-dbg_2.02-4build1_amd64.deb     && apt-get remove -y --purge software-properties-common         && apt-get autoremove -y         && apt-get clean         && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*' returned a non-zero code: 100
Stopping porter_nginx_1 ... done
```

**Comment by Keoghan** [comment]
think we might need to tweak the packages slightly.

https://askubuntu.com/a/1143014

**Comment by Keoghan** [comment]
started work in #64

**Comment by rdarcy1** [comment]
Mirrors for those package versions are no longer active, using repository given for the Java port of pdftk: https://gitlab.com/pdftk-java/pdftk

```
sudo add-apt-repository ppa:malteworld/ppa
sudo apt update
sudo apt install pdftk
```

Official pdftk-java package is available for Ubuntu >=18.10 when we upgrade the base image.
